resources/v1/domains/testing/fixtures.go

```diff
@@ -250,13 +21,15 @@ func HandleListDomainsSuccessfully(t *testing.T) {
 		w.Header().Set("Content-Type", "application/json")
 		w.WriteHeader(http.StatusOK)
 
+		fixtureName := "list.json"
 		if (r.URL.Query().Get("service") == "shared" || r.URL.Query().Get("area") == "shared") &&
 			r.URL.Query().Get("resource") == "things" {
-			th.TestHeader(t, r, "X-Limes-Cluster-Id", "fakecluster")
-			fmt.Fprintf(w, domainFilteredListJSON)
+			fixtureName = "list-filtered.json"
 		}
 
-		fmt.Fprintf(w, domainListJSON)
+		jsonBytes, err := os.ReadFile(filepath.Join("fixtures", fixtureName))
+		th.AssertNoErr(t, err)
+		fmt.Fprint(w, string(jsonBytes))
```

Comment:
This can be simplified to
```suggestion
		w.Write(jsonBytes)
```